style.css

```diff
@@ -437,4 +437,382 @@ input[type=email] {
   .fontawesome {
     display: none;
   }
+
+  .uber-navig {
+    display: flex;
+    flex-direction: column;
+    width: 345px;
+    height: 596px;
+    padding: 16px;
+    margin: 25px 24px;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    background-color: #fff;
+    justify-content: center;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    width: 1600px;
+  }
+
+  .header {
+    height: 280px;
+    background-color: #fff;
+    padding: 25px 155px;
+  }
```

Comment:
- [ ] Add some missing menus at the top. (Contact, Skills). Check for more details on the Figma.
![menu1](https://user-images.githubusercontent.com/55635977/126771993-2e22302b-ba1d-4f1c-9e90-f12ffed7eab8.png)